Makefile.generated_files

```diff
@@ -253,20 +253,18 @@ $(DEEPCOPY_FILES): $(DEEPCOPY_GEN)
 # file if the contents have actually changed.  We 'sinclude' this later.
 .PHONY: $(META_DIR)/$(DEEPCOPY_GEN).mk
 $(META_DIR)/$(DEEPCOPY_GEN).mk:
-	mkdir -p $(@D);                                        \
-	(echo -n "$(DEEPCOPY_GEN): ";                          \
-	 DIRECT=$$(go list -f '{{.Dir}} {{.Dir}}/*.go'         \
-	     ./cmd/libs/go2idl/deepcopy-gen);                  \
-	 INDIRECT=$$(go list                                   \
-	     -f '{{range .Deps}}{{.}}{{"\n"}}{{end}}'          \
-	     ./cmd/libs/go2idl/deepcopy-gen                    \
-	     | grep --color=never "^$(PRJ_SRC_PATH)"           \
-	     | sed 's|^$(PRJ_SRC_PATH)|./|'                    \
-	     | xargs go list -f '{{.Dir}} {{.Dir}}/*.go');     \
-	 echo $$DIRECT $$INDIRECT                              \
-	     | sed 's/ / \\=,/g'                               \
-	     | tr '=,' '\n\t';                                 \
-	) | sed "s|$$(pwd -P)/||" > $@.tmp;                    \
+	mkdir -p $(@D);                                                       \
+	(echo -n "$(DEEPCOPY_GEN): ";                                         \
+	 go list                                                              \
+	     -f '{{.ImportPath}}{{"\n"}}{{range .Deps}}{{.}}{{"\n"}}{{end}}'  \
+	     ./cmd/libs/go2idl/deepcopy-gen                                   \
+	     | grep --color=never "^$(PRJ_SRC_PATH)/"                         \
+	     | xargs go list -f '{{.Dir}} {{.Dir}}/*.go'                      \
+	     | paste -sd' ' -                                                 \
+	     | sed 's/ / \\=,/g'                                              \
+	     | tr '=,' '\n\t'                                                 \
+	     | sed "s|$$(pwd -P)/||";                                         \
```

Comment:
Ideally we'd just push folks to use the dockerized build.  If we do try and make mac work, I prefer we look to use `gsed` if `sed` isn't right.  This is the default for homebrew and helps to avoid other issues by installing gnu sed.